ports/esp32/boards/ESP32_GENERIC_C2/board.md

```diff
@@ -0,0 +1,3 @@
+The following files are firmware images that should work on most
+ESP32-C2-based boards with 4MiB of flash and 26MHz crystal frequency.
```

Comment:
Maybe worth saying "with at least 4MiB of flash"?

Reply:
I changed this because currently all ESP8684 chips have either 2MB or 4MB of embedded flash, and I actually thought the flash controller only supports those sizes. But it looks like it can at least theoretically support more.